apps/condo/domains/notification/adapters/firebaseAdapter.js

```diff
@@ -0,0 +1,200 @@
+const admin = require('firebase-admin')
+const isEmpty = require('lodash/isEmpty')
+const isNull = require('lodash/isNull')
+const get = require('lodash/get')
+const faker = require('faker')
+
+const {
+    PUSH_FAKE_TOKEN_SUCCESS,
+    PUSH_FAKE_TOKEN_FAIL,
+    FIREBASE_CONFIG_ENV,
+} = require('../constants/constants')
+
+const EMPTY_CONFIG_ERROR = `Valid ${FIREBASE_CONFIG_ENV} should be provided within .helm (.env), and can be retrieved from https://console.firebase.google.com/project/__PROJECT_ID__/settings/serviceaccounts/adminsdk`
+const EMPTY_NOTIFICATION_TITLE_BODY_ERROR = 'Missing notification.title or notification.body'
+
+let FIREBASE_CONFIG = null
+
+console.error('process.env.GOOGLE_RECAPTCHA_CONFIG.length:', get(process.env, 'GOOGLE_RECAPTCHA_CONFIG.length', undefined))
+
+if (!process.env[FIREBASE_CONFIG_ENV]) throw new Error(EMPTY_CONFIG_ERROR)
+
+// JSON.parse would throw for a broken JSON, so we should take care of it here
+try {
+    FIREBASE_CONFIG = JSON.parse(process.env[FIREBASE_CONFIG_ENV])
```

Comment:
it's better to use `conf. FIREBASE_CONFIG`

Reply:
done